src/yoroi-wallets/cardano/ShelleyWallet.test.ts

```diff
@@ -101,10 +101,10 @@ describe('migration', () => {
       `"stake1u8q3auyvgnekzzm72m2xuzrtjqvxcyhf568s2gdhcnrjujcr25kkv"`,
     )
 
-    expect(wallet.transactionCache?.perRewardAddressCertificates).toMatchInlineSnapshot(`Object {}`)
-    expect(wallet.transactionCache?.confirmationCounts).toMatchInlineSnapshot(`Object {}`)
-    expect(wallet.transactionCache?.perAddressTxs).toMatchInlineSnapshot(`Object {}`)
-    expect(wallet.transactionCache?.transactions).toMatchInlineSnapshot(`Object {}`)
+    expect(wallet.transactionManager.perRewardAddressCertificates).toMatchInlineSnapshot(`Object {}`)
```

Comment:
these were migration tests
you can get rid of anything testing private wallet values